From 790c73ad93732d31ec6ae5ba4d7116d7f8fb46a2 Mon Sep 17 00:00:00 2001
From: Kalev Lember <kalevlember@gmail.com>
Date: Sat, 10 Aug 2013 14:54:04 +0200
Subject: [PATCH] minizip: Add missing include

Include zconf.h for z_crc_t, fixes a regression from commit
63ba7582b80eb81b126c2931e485481c35596aab.
---
 contrib/minizip/crypt.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/contrib/minizip/crypt.h b/contrib/minizip/crypt.h
index 1e9e820..835a2ba 100644
--- a/crypt.h
+++ b/crypt.h
@@ -27,6 +27,8 @@
    Encryption is not supported.
 */
 
+#include "zconf.h"
+
 #define CRC32(c, b) ((*(pcrc_32_tab+(((int)(c) ^ (b)) & 0xff))) ^ ((c) >> 8))
 
 /***********************************************************************
-- 
1.8.3.1

